pkg/diff/diff.go

```diff
@@ -406,6 +417,11 @@ func (sc *Syncer) eventLoop(ctx context.Context, d Do) error {
 }
 
 func (sc *Syncer) handleEvent(ctx context.Context, d Do, event crud.Event) error {
+	// Skip licenses if includeLicense is disabled.
+	// TODO: use a clearer or more general way to ignore licenses when includeLicenses=false?
+	if !sc.includeLicenses && event.Kind == KindLicense {
+		return nil
+	}
```

Comment:
What actually happens if this is absent?

Convention (sorta, there's only one other skippable entity) is to skip [them in the state builder](https://github.com/Kong/go-database-reconciler/blob/fa395292411a55869e7e742f27f6e232eb492984/pkg/file/builder.go#L86-L88), which I think should be equivalent to skipping events (rather, it'd not generate them in the first place).

The earlier proposal to add full license entity support [only included the syncer skip](https://github.com/Kong/deck/pull/882/files#diff-61c9587f2d7a5abbf52f69e0983b087ec5309c08721a20bc4155f57e1119fb6eR267-R276), though I don't recall why it didn't have a builder skip similar to CA certificates--it may well have just been something I should have had, but overlooked.

Reply:
If we do not skip the event here, the event will be called by the `Do` func in `d(event)`. I will see that if skip them in the builder works.